docs/understand/texture_fetching.rst

```diff
@@ -0,0 +1,107 @@
+.. meta::
+  :description: This chapter describes the texture fetching modes of the HIP ecosystem
+  ROCm software.
+  :keywords: AMD, ROCm, HIP, Texture, Texture Fetching
+
+*******************************************************************************
+Texture Fetching
+*******************************************************************************
+
+`Textures <https://rocm.docs.amd.com/projects/HIP/en/latest/doxygen/html/group___texture.html>`_ are more than just a buffer, that is interpreted as 1D, 2D or 3D array. Because of its legacy as a graphics functionality, textures are indexed with floats. This can happen two different ways, either the index will be in the range of [0..size-1] or in [0..1]. The difference is mathematically just a division, so for the explanations on this page, we'll use the [0..size-1] indices.
```

Comment:
Almost overlooked it: this hyperlink currently always redirects to "latest", even if the user is looking at an older version of the documentation. This should hopefully do the trick
```suggestion
[Textures](../doxygen/html/group___texture) are more than just a buffer, that is interpreted as 1D, 2D or 3D array. Because of its legacy as a graphics functionality, textures are indexed with floats. This can happen two different ways, either the index will be in the range of [0..size-1] or in [0..1]. The difference is mathematically just a division, so for the explanations on this page, we'll use the [0..size-1] indices.
```

Reply:
Whoops, seems like I messed up. `../../doxygen/html/group___texture.html` as a link should fix it :)

Reply:
checked it. `../doxygen/html/group___texture.html` seems to be the one working

Reply:
Nope, it still needs the additional `../`, otherwise it just reaches the `understand` directory

Reply:
well, it works now. and it doesn't work with the second `../` so i guess i'll leave it like this

Reply:
Weird, yesterday the link still rendered wrongly. Works for me now too.